### PR TITLE
Toggle subsidiary

### PR DESCRIPTION
**Pull request by CatoLeanTruetschel**

### CODE REVIEW

**Review by PianoRollRepresentation — APPROVED** [review]
LGTM

File: frontend/src/visualization/filtering/SubsidiaryNodesToggle.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      <FormControlLabel
+        control={
+          <Checkbox
+            checked={includeSubsidiary}
+            onChange={(_, val) => updateIncludeSubsidiary(val)}
+            name="show-subsidiary"
+            color="primary"
+          />
+        }
+        label="Show subsidiary"
```
Comment (joluj) [inline frontend/src/visualization/filtering/SubsidiaryNodesToggle.tsx:32]: Is subsidary a common graph term? If not the description from the would be really helpful. You could show that for example with a [tooltip](https://material-ui.com/components/tooltips/).

**Review by joluj — APPROVED** [review]
LGTM

File: frontend/src/visualization/filtering/SubsidiaryNodesToggle.tsx (hunk 1 above)
Comment (CatoLeanTruetschel) [reply frontend/src/visualization/filtering/SubsidiaryNodesToggle.tsx:32]: No it's not a common term. The explanation in the UI is tracked by #256

**Pull request merged**